.github/workflows/mark-stale-prs.yaml

```diff
@@ -1,7 +1,7 @@
-name: "Close stale pull requests"
+name: 'Close stale pull requests'
```

Comment:
Not important, but I see we now have [names](https://github.com/adobecom/milo/blob/stage/.github/workflows/merge-to-stage.yaml#L1) without any quotes, [names](https://github.com/adobecom/milo/blob/stage/.github/workflows/mark-stale-prs.yaml#L1) with double quotes and now this. We should stick to a rule for consistency

Reply:
There is no easy was to decide on this, for longer text with special characters, using (single) quotes is the way to go.
This won't work with numbers though, as it would convert numbers to strings.
Then there are smaller lines such as
```
    runs-on: ubuntu-latest
```

...

For the sake of simplicity, I'll just remove the quotes from the name for now and keep it on the cron schedule

Reply:
If we don't like the inconsistency we should do _one_ PR and potentially look into configuring the linter to handle a standard config for us in the future